k8s/namespaces/admin/aad-pod-identity/patches/sbox/aad-pod-id.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nmi
+spec:
+  template:
+    spec:
+      containers:
+      - name: nmi
+        env:
+        - name: FORCENAMESPACED
+          value: "true"
+      tolerations:
+      - effect: NoSchedule
+        key: CriticalAddonsOnly
+        operator: Equal
+        value: "true"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mic
+spec:
+  template:
+    spec:
+      containers:
+      - name: mic
+        env:
+        - name: FORCENAMESPACED
+          value: "true"
+      tolerations:
+      - effect: NoSchedule
+        key: CriticalAddonsOnly
+        operator: Equal
+        value: "true"
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 1
+            preference:
+              matchExpressions:
+              - key: agentpool
```

Comment:
isn't it a full domain key not just the text agentpool ?

i.e. we should probably use the label and not the agentpool name

Reply:
```suggestion
              - key: kubernetes.azure.com/mode
```

Reply:
yeah I started doing that then got distracted 🤦🏼, will do the rest now

Reply:
good spot, updated 👍🏼